### PR TITLE
Fix issue ticket 1 - please review and improve my try!

### PR DESCRIPTION
**Pull request by emma-martin**
Hi @stephanieggullo , please check my try at solving ticket 1. 

See comments below!

Thanks!!!

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -13,4 +13,11 @@
     top: 50%;
     transform: translate(-50%, -60%) rotate(90deg);
   }
+
+  .page {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+  }
```
Comment (emma-martin) [inline main.css]: Here I used the old display flex trick, using justify content and align items, but it only aligns itself horizontally, not vertically, across the page div. That's why I used the height property, to give some more room for flex to work properly.

What do you think? Would have you done something different?

Comment (stephanieggullo) [reply main.css]: maybe we can try without a new div (page), using a height body.

Comment (emma-martin) [reply main.css]: Done! Please check how it looks now 🥇

Comment (stephanieggullo) [reply main.css]: perfect :+1:

**Pull request merged**